<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <widget class="QWidget" name="widget_7" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_6">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QVBoxLayout" name="verticalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="widget" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_2">
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QTextBrowser" name="msgBrowser"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_2">
                 <property name="frameShape">
                  <enum>QFrame::Box</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="leftMargin">
                   <number>3</number>
                  </property>
                  <property name="topMargin">
                   <number>3</number>
                  </property>
                  <property name="rightMargin">
                   <number>3</number>
                  </property>
                  <property name="bottomMargin">
                   <number>3</number>
                  </property>
                  <item>
                   <widget class="QFontComboBox" name="fontCbx"/>
                  </item>
                  <item>
                   <widget class="QComboBox" name="sizeCbx">
                    <property name="currentText">
                     <string>12</string>
                    </property>
                    <property name="currentIndex">
                     <number>7</number>
                    </property>
                    <item>
                     <property name="text">
                      <string>5</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>6</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>7</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>8</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>9</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>10</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>11</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>12</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>13</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>14</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>15</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>16</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>17</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>18</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>19</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>20</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="boldTBtn">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="toolTip">
                     <string>加粗</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="ref.qrc">
                      <normaloff>:/img/bold.png</normaloff>:/img/bold.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="italicTBtn">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="toolTip">
                     <string>倾斜</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="ref.qrc">
                      <normaloff>:/img/italic.png</normaloff>:/img/italic.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="underlineTBtn">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="toolTip">
                     <string>下划线</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="ref.qrc">
                      <normaloff>:/img/under.png</normaloff>:/img/under.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="colorTBtn">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="toolTip">
                     <string>更改字体颜色</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="ref.qrc">
                      <normaloff>:/img/color.png</normaloff>:/img/color.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="saveTBtn">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="toolTip">
                     <string>保存聊天记录</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="ref.qrc">
                      <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QToolButton" name="clearTBtn">
                    <property name="minimumSize">
                     <size>
                      <width>26</width>
                      <height>26</height>
                     </size>
                    </property>
                    <property name="toolTip">
                     <string>清空聊天记录</string>
                    </property>
                    <property name="text">
                     <string>...</string>
                    </property>
                    <property name="icon">
                     <iconset resource="ref.qrc">
                      <normaloff>:/img/clear.png</normaloff>:/img/clear.png</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>20</width>
                      <height>20</height>
                     </size>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_3" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>100</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTextEdit" name="msgTxtEdit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>100</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="usrName">
              <property name="text">
               <string>用户：xxx</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="sendBtn">
              <property name="text">
               <string>发送</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+Return</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="usrNumLbl">
              <property name="text">
               <string>在线人数：0人</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="exitBtn">
              <property name="text">
               <string>退出</string>
              </property>
              <property name="shortcut">
               <string>Ctrl+Q</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QTableWidget" name="usrTblWidget">
           <property name="maximumSize">
            <size>
             <width>180</width>
             <height>16777215</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>用户名</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ref.qrc"/>
 </resources>
 <connections/>
</ui>
